learn/getting_started/cloud_quick_start.mdx

```diff
@@ -0,0 +1,81 @@
+---
+imageURL: https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/cloud-getting-started
+---
+
+# Getting started with Meilisearch Cloud
+
+This tutorial walks you through setting up Meilisearch Cloud, creating a project and an index, adding documents to it, and performing your first search with the default web interface.
+
+You need a Meilisearch Cloud account to follow along. If you don't have one, register for a 14-day free trial account at [https://cloud.meilisearch.com/register](https://cloud.meilisearch.com/register).
+
+## Creating a project
+
+To use Meilisearch Cloud, you must first create a project. Projects act as containers for indexes, tasks, billing, and other information related to Meilisearch Cloud.
+
+Click the "New project" button on the top menu. If you have a free trial account and this is your first project, the button will read "Start free trial" instead:
+
+![The Meilisearch Cloud menu, featuring the "New Project" button](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/1-new-project.png)
+
+Name your project `meilisearch-quick-start` and select the region closest to you, then click on "Create project":
+
+![A modal window with two mandatory fields: "Project name" and "Select a region"](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/2-create-project.png)
+
+If you are not using a free trial account, you must also choose a billing plan based on the size of your dataset and number of searches per month:
+
+![A variation of the previous modal window with an extra mandatory field: "Select a plan". There are four billing plan options, from cheapest to most expensive: "Build", "Grow", "Pro", and "Enterprise"](https://raw.githubusercontent.com/meilisearch/documentation/cloud-quick-start/assets/images/cloud-getting-started/3-project-billing.png)
+
+Creating your project might take a few minutes. Once it is ready, you will see it in the project list. Click on the project name to go to the project overview page:
```

Comment:
Maybe you want to be more precise here, not sure.

The project will be in the project list from the moment the user creates it.
When it's ready, there will be an indicator ✅ on the right of the card, or a "ready" indication on the project overview.